internal/command/local/printers.go

```diff
@@ -238,6 +240,42 @@ func getRawPortforward(cfg *config.LocalStatus, ciConfig *config.ConnectInvocati
 	return result
 }
 
+func getRawWatcher(cfg *config.LocalStatus, watcher *commonapi.WatcherStatus,
+	statusMap map[string]any) any {
+	var result any
+
+	if cfg.Details {
+		// Details view
+		result = statusMap["watcher"]
```

Comment:
can we just return `statusMap["watcher"]` here?

Reply:
It's done, I took the same approach in the other related functions.